### PR TITLE
attempt submission

### PR DESCRIPTION
**Pull request by tkcy**

### CODE REVIEW

File: model-metadata/CU-RSV_SVIRS.yml
Comment (kjsato) [inline model-metadata/CU-RSV_SVIRS.yml:1]: Thanks for your submission!

Could you add the lines below in your model-metadata?

1. schema version (please add it in the first line as is:)
schema_version: "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.
0/admin-schema.json"

2. website_url
it is needed as a required property like:
website_url: "https://url_to_yourmodel_website.com"

Comment (kjsato) [inline model-metadata/CU-RSV_SVIRS.yml:1]: Thanks for your submission!

Could you add the lines below in your model-metadata?

schema version (please add it in the first line as is:)
schema_version: "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.
0/admin-schema.json"

website_url
it is needed as a required property like:
website_url: "https://url_to_yourmodel_website.com"

Comment (tkcy) [reply model-metadata/CU-RSV_SVIRS.yml:1]: updated!

**Comment by tkcy** [comment]
@shauntruelove No worries, I changed the date.  Also added a space after a ":", hopefully that solves the metadata issue?  

You're probably aware but just in case you're not, the guidelines in the word doc says the origin date is the Saturday following the submission Due Date, in case you want to update that.

**Comment by kjsato** [comment]
@tkcy As for metadata, could you chenge the license line as:

license: "CC-BY-4.0"

sorry for the inconvenience, but this should resolve the error on model-metadata

**Comment by tkcy** [comment]
@kjsato trying again

**Comment by kjsato** [comment]
@tkcy thanks! and sorry for the inconvenience. I updated [tasks.json in main](https://github.com/HopkinsIDD/rsv-forecast-hub/blob/main/hub-config/tasks.json), could you merge or replace it in your `tkcy:main`? 

Then I think the submission will pass if you can resolve the mismatches that appeared in the Notice line above.

**Comment by tkcy** [comment]
Hrm so we had this problem with the origin datetime format for our RSV scenario hub submission - my computer somehow adds time zone info that is not visible on my end. it was solved there by converting to character format. Maybe I'll try going back to character.

**Comment by kjsato** [comment]
> Hrm so we had this problem with the origin datetime format for our RSV scenario hub submission - my computer somehow adds time zone info that is not visible on my end. it was solved there by converting to character format. Maybe I'll try going back to character.

Coud you use below for example before saving the data without changing the type "Date"? 
data1$origin_date <- as.Date(data1$origin_date)

**Comment by tkcy** [comment]
We use Matlab, not R, so there are differences in translation.

I'm trying again here, with the date modified.  It also looks like one of the error messages says point forecasts are invalid. 
 Should I remove those?

**Comment by kjsato** [comment]
The attached parquet is the revision which I made converted the origin_date column. Just FYI(gzipped to attach here [2024-02-25-CU-RSV_SVIRS.parquet.gz](https://github.com/HopkinsIDD/rsv-forecast-hub/files/14453454/2024-02-25-CU-RSV_SVIRS.parquet.gz)). By using it, only one remaining issue is:
```
✖ 2024-02-25-CU-RSV_SVIRS.parquet: `tbl` contains invalid values/value combinations.  Column `output_type_id` contains
  invalid values "0.15000000000000002", "0.35000000000000003", "0.49999999999999994", "0.5499999999999999",
  "0.5999999999999999", "0.6499999999999999", "0.7999999999999999", and "0.8999999999999999".
```
Please take a look at the contents and see if there is a possibility of correction

**Comment by tkcy** [comment]
I don't know how to resolve - I'm entering as specified but the precision of doubles in Matlab is 16 decimal places. I would suggest accepting this column as a character array.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Koji SATO ***@***.***>
Sent: Thursday, February 29, 2024 3:56:39 PM
To: HopkinsIDD/rsv-forecast-hub ***@***.***>
Cc: Yamana, Teresa K. ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [HopkinsIDD/rsv-forecast-hub] attempt submission (PR #7)


The attached parquet is the revision which I made converted the origin_date column. Just FYI(gzipped to attach here 2024-02-25-CU-RSV_SVIRS.parquet.gz<https://github.com/HopkinsIDD/rsv-forecast-hub/files/14453454/2024-02-25-CU-RSV_SVIRS.parquet.gz>). By using it, only one remaining issue is:

✖ 2024-02-25-CU-RSV_SVIRS.parquet: `tbl` contains invalid values/value combinations.  Column `output_type_id` contains
  invalid values "0.15000000000000002", "0.35000000000000003", "0.49999999999999994", "0.5499999999999999",
  "0.5999999999999999", "0.6499999999999999", "0.7999999999999999", and "0.8999999999999999".


Please take a look at the contents and see if there is a possibility of correction

**Comment by tkcy** [comment]
Thanks, that looks good.

**Comment by kjsato** [comment]
okay, I will proceed with it

**Pull request merged**